docs/riff_streaming.md

```diff
@@ -8,7 +8,10 @@ title: "riff streaming"
 
 ### Synopsis
 
-<todo>
+The streaming runtime uses riff Functions, Processor and Stream custom resources to deploy streaming workloads.
```

Comment:
All lines need to wrap at 80 characters since this is rendered to a terminal

Reply:
Fixed. What about having `riff docs` generate properly wrapped docs?

Reply:
or even a test could introspect all commands Long and Short descriptions and fail if width > 80.

Reply:
Test added: https://github.com/projectriff/cli/pull/194/commits/ab0aa53c7d109b6771fc3a16e8e2547ea498a54c#diff-cb3cf3b1238fa22cdc8ed6e77c49b4e0R89
and culprits fixed.